app/design/frontend/base/default/template/securesubmit/form.phtml

```diff
@@ -1,165 +1,141 @@
 <?php /* @var $this Hps_Securesubmit_Block_Form */
 $_code = $this->getMethodCode();
+$_loggedIn = Mage::getSingleton('customer/session')->isLoggedIn();
 $public_key = Mage::getModel('hps_securesubmit/payment')->getConfigData('publicapikey');
+$token_data_url = Mage::getUrl('securesubmit/storedcard/getTokenData', array('_secure' => TRUE));
 $allow_card_saving = Mage::getModel('hps_securesubmit/payment')->getConfigData('allow_card_saving');
 $allow_heartland_gift = Mage::getModel('hps_securesubmit/payment')->getConfigData('allow_heartland_gift');
-$_loggedIn = Mage::getSingleton('customer/session')->isLoggedIn();
+$use_iframes = !!Mage::getModel('hps_securesubmit/payment')->getConfigData('use_iframes');
```

Comment:
Yes, it's stored as a bit. 

> On Jun 6, 2016, at 18:06, Charlie Simmons <notifications@github.com> wrote:
> 
> In app/design/frontend/base/default/template/securesubmit/form.phtml:
> 
> >  $allow_card_saving = Mage::getModel('hps_securesubmit/payment')->getConfigData('allow_card_saving');
> >  $allow_heartland_gift = Mage::getModel('hps_securesubmit/payment')->getConfigData('allow_heartland_gift');
> > -$_loggedIn = Mage::getSingleton('customer/session')->isLoggedIn();
> > +$use_iframes = !!Mage::getModel('hps_securesubmit/payment')->getConfigData('use_iframes');
> not not? Are you just trying to force a bool true here?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>